lightning/src/onion_message/messenger.rs

```diff
@@ -25,6 +25,8 @@ use crate::ln::msgs::{self, OnionMessage, OnionMessageHandler, SocketAddress};
 use crate::ln::onion_utils;
 use crate::routing::gossip::{NetworkGraph, NodeId, ReadOnlyNetworkGraph};
 use super::async_payments::AsyncPaymentsMessageHandler;
+#[cfg(async_payments)]
```

Comment:
Looks like this needs to move to the last commit?

Reply:
Oops, thanks for catching that